Skuid_Techniques/README.md

```diff
@@ -6,7 +6,6 @@ We know its true - some of the coolest capabilities provided by Skuid are compli
 - [Adding related records to drawers](Drawers_Related_Records)
 - [Advanced Filtering Techniques](Filters_Using_Forms)
 - [Key Indicators Dashboard](KeyIndicators)
-- [Navigation - Advanced Tehcniques](Navigation)
```

Comment:
@Rnhatch  Did you mean to delete this line as part of this PR?

Reply:
I did not.  Let me correct.  Here comes a push